README.md

```diff
@@ -15,7 +15,37 @@ To do this, open
 ## Usage
 After installing this plugin and the core ROSIntegration plugin, you can load your UE4 project.
 If the plugin is not enabled in the project yet, you can do this in the UE4Editor in Edit -> Plugins.
-When the plugin is loaded correctly, you should be able to add a new Actor called "Vision Actor" to your level, which represents the RGBD camera. In some cases (for example on Linux), it might be necessary to call the Generate Project Files action on UE4 in order to fetch the new header files for the plugin before your first compile.
+
+When the plugin is loaded correctly, a new `CameraActor` named `VisionActor` and a new `CameraComponent` named `VisionComponent` will be available to use.
+Each one represents the RGBD camera.
+
+In some cases (for example on Linux), it might be necessary to call the Generate Project Files action on UE4 in order to fetch the new header files for the plugin before your first compile. 
+
+### Vision Component
+
+Actor Attachment:
+
+```c++
+#include "ROSIntegrationVision/Public/VisionComponent.h"
+...
+UVisionComponent * vision = CreateDefaultSubobject<UVisionComponent>(TEXT("Vision"));
+vision->SetupAttachment(RootComponent);
+```
+
+Fixed Camera Mode:
+
+This disables tf publishing from within Unreal, and instead requires the links 
+to be specified via URDF.  
+
+```c++
+vision->FixedCam = true;
```

Comment:
I think the variable name 'FixedCam' is a bit misleading. The only function of this variable right now is disable tf publish. So i guess it could also be named 'DisableTFPublishing'